modules/@apostrophecms/page-type/index.js

```diff
@@ -53,13 +58,56 @@ module.exports = {
       }
     };
   },
+  columns() {
+    return {
+      add: {
+        title: {
+          name: 'title',
+          label: 'apostrophe:title',
+          component: 'AposCellButton'
+        },
+        slug: {
+          name: 'slug',
+          label: 'apostrophe:slug',
+          component: 'AposCellButton'
+        },
+        updatedAt: {
+          name: 'updatedAt',
+          label: 'apostrophe:lastEdited',
+          component: 'AposCellLastEdited'
+        }
+      }
+    };
+  },
+  filters: {
+    add: {
+      archived: {
```

Comment:
only the archived should be needed, not the visibility, if we want to be consistent with the regular page manager (ie: the page tree)